tests/e2e/volume_access_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The goal of this test suite is to verify read/write consistency on volumes
+// in accordance with the access updates on the volume
+package e2e
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/vmware/docker-volume-vsphere/tests/utils/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/inputparams"
+)
+
+const ErrorWriteVolume = "Read-only file system"
+
+type VolumeAccessTestSuite struct {
+	volumeName    string
+	dockerHostIP  []string
+	containerList [2][]string
+	esxIP         string
+}
+
+func (s *VolumeAccessTestSuite) SetUpSuite(c *C) {
+	s.dockerHostIP = []string{os.Getenv("VM1"), os.Getenv("VM2")}
+	s.esxIP = os.Getenv("ESX")
+
+	dsName := govc.GetDatastoreList()[0]
```

Comment:
Sure. Will add the comment.. This can be any datastore. There should be atleast one (where the vm resides). The name can be datastore1 in case of CI, or anything else.